temboardui/templates/home.html

```diff
@@ -65,6 +65,7 @@
         :class="['card mb-2 mr-2', {'border-danger bg-danger-light': getStatusValue(instance) >= 1000000, 'border-warning bg-warning-light': getStatusValue(instance) >= 1000}]"
         style="width: 220px;"
         v-for="instance in filteredInstances"
+        :key="instance.hostname"
```

Comment:
This `key` should be unique, isn't it ?
I think we could concatenate `instance.hostname` with `instance.pg_port` for unicity.